src/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -82,6 +82,9 @@
 import java.util.Map;
 import java.util.Set;
 
+import static java.util.Arrays.*;
+import static java.util.Collections.*;
+import static org.javarosa.core.model.Constants.*;
```

Comment:
I like the wildcard import for Constants (because there are so many), but somewhat dislike it here for Arrays and Collections.

Reply:
Yeah... static imports on Collections are kind of nice, though...